createUser/function.json

```
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "name": "createUserMessage",
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "user-queue",
+      "connection": "radarstrg_STORAGE"
+    }
+  ],
+  "scriptFile": "../dist/createUser/index.js"
```

```suggestion
  "scriptFile": "../dist/CreateUser/index.js"
```
Change folder's name as well. (To do so, commit twice- one without the folder, and one with the uppercase name)